### DISCUSSION TITLE
manage the volumetric weight and more

### DISCUSSION THREAD

**Discussion by ballashop** [PrestaShop/PrestaShop#40198] [Ideas]
Some couriers put limits on measures. Not only from the amount of the 3 lengths, already available on prestashop.
Could you add a field like the one in image?
![more-field](https://user-images.githubusercontent.com/28685316/70062290-0c39d700-15e6-11ea-85b7-5edc8281e237.gif)

**Weight / volume management.**

At the moment prestashop manages just weight ranges, while most carriers works with weight / volume one.
In prestashop it is not possible to create zones by volume weight.
Are you thinking about this feature?
Thanks a lot for your support.
Emiliano

**Comment by khouloudbelguith** [comment]
Hi @ballashop,

Thanks for your report.
Ping @PrestaShop/prestashop-product-team what do you think?

Thanks!

**Comment by HartLarsson** [comment]
could be more easy to implement a new field cm3/kg or m3/kg where you can put the parameter so prestashop can calculate (LxAxW)/new-field to calculate the volumetric weight.
If the volumetric weight is > of the weight specified inside the product the columetric weight will be used to calculate the shipping price inside the currier table.

**Comment by Hlavtox** [comment]
This is a real need, but it differs from carrier to carrier. Some define longest side. Some define length+width+height. Some define 2xlength+2xwidth+2xheight. The more code we bundle to the software, more burden to maintain.

I think the best would be to let people implement it via a hook. If an example is not available, we can create it.

cc @PrestaShop/product-council
